We do not need to test correctness of the README file using rst2html.py.
Also, the correct name of the file is README.rst, not README.

--- testresources-2.0.1/tox.ini.orig
+++ testresources-2.0.1/tox.ini
@@ -5,4 +5,3 @@
 deps = .[test]
 commands =
     python -m testtools.run discover
-    rst2html.py --strict README /dev/null
